lib/routes/charge/items/item_avatar.dart

```diff
@@ -68,6 +75,46 @@ Color fromHex(String hexString) {
   return Color(int.parse(buffer.toString(), radix: 16));
 }
 
+class _IconAvatar extends StatelessWidget {
+  final double radius;
+  final String iconName;
+  final bool useDecoration;
+
+  _IconAvatar(this.radius, this.iconName, this.useDecoration);
+
+  @override
+  Widget build(BuildContext context) {
+    Map<String, IconData> iconMapping = IconMap().iconMap;
+    String iconName = this.iconName.substring(5);
+    return Container(
+      width: radius * 2,
+      height: radius * 2,
+      decoration: useDecoration
+          ? BoxDecoration(
+              shape: BoxShape.circle,
+              border: Border.all(
+                  color: Colors.white, width: 1.0, style: BorderStyle.solid),
+              image: DecorationImage(
+                  colorFilter: ColorFilter.mode(
+                      Theme.of(context).primaryColorLight, BlendMode.srcATop),
+                  image: AssetImage("src/images/avatarbg.png"),
+                  fit: BoxFit.cover))
+          : null,
+      child: Column(
+        mainAxisSize: MainAxisSize.max,
+        mainAxisAlignment: MainAxisAlignment.center,
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: <Widget>[
+          Icon(
+            iconMapping[iconName],
+            size: useDecoration ? radius : radius * 1.5,
```

Comment:
Why do we duplicate the radius at 1.5 ?

Reply:
We want the icon to look bigger on catalog list without making the container larger.